packages/tests/temporal/src/entities/Author.test.ts

```diff
@@ -0,0 +1,75 @@
+import { newEntityManager } from "@src/setupDbTests";
+import { jan1, jan2, jan3 } from "@src/utils";
+import { PrimitiveField, alias, getMetadata } from "joist-orm";
+import { Temporal } from "temporal-polyfill";
+import { Author, Book, newAuthor, newBook } from "./entities";
+
+describe("Author", () => {
```

Comment:
I did a lot of "put tests into files named after entities" but lately have been trying to name the test files after the feature, so maybe do like PlainDate.test + DateTime.test + ZonedDateTime.test? Or just one `Temporal.test.ts` if you want to combine them all.